themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig

```diff
@@ -60,6 +60,11 @@
           <h3 class="banner-title">
             {{ content.field_prgf_headline }}
           </h3>
+          {% if content.field_prgf_description.0['#text'] is not empty %}
```

Comment:
We should set variable like here https://github.com/ymcatwincities/openy/pull/393/files#diff-099c1735c766cb3803f7f2f2bca786d4R48. We should never use array properties in twig templates. If we need to do this, we should define new variables above and then use them in conditions.